newIDE/app/src/GameDashboard/Feedbacks/FeedbackCard.js

```diff
@@ -157,18 +157,28 @@ const FeedbackCard = ({
                       )}
                     </Text>
                   )}
-                  <BackgroundText style={styles.backgroundText} allowSelection>
-                    {comment.playerId ? (
-                      <Link
-                        onClick={() => setOpenPlayerPublicProfileDialog(true)}
-                        href="#"
-                      >
+                  {comment.playerId ? (
+                    <Link
+                      onClick={() => setOpenPlayerPublicProfileDialog(true)}
+                      href="#"
+                    >
+                      <Text noMargin color="inherit">
```

Comment:
This could even be a badge showing the avatar of the player user account, but I guess this is something to be thought about in the app in general (whenever we show a user profile badge, like on examples, we should show the avatar)

Reply:
(and this avatar should be made so that it displays a blank skeleton or a generic image while the avatar image is not loaded - so it's not entirely trivial to make it well... but important for the future)